packages/server/src/script/resendSignupEmails.ts

```diff
@@ -0,0 +1,136 @@
+        const file = storageFileFromId(info.id.concat('-') + numResends)
+        await file.upload(pdfBuffer)
+
+        // Send faxes
+        let twilioResponses: TwilioResponse[] = []
+        if (method.faxes.length > 0) {
+            const uri = await file.getSignedUrl(24 * 60 * 60 * 1000)
+            const resposnes = await sendFaxes(uri, method.faxes)
+            twilioResponses = resposnes.map(({url, sid, status}) => ({url, sid, status}))
+
+            for (const twilioResponse of twilioResponses) {
+                if (twilioResponse.status != "queued") {
+                    console.log('Error sending faxes for ' + info.id + '. Exiting.')
+                    console.log(twilioResponse)
+                    return
+                }
+            }
+          }
+
+        // Update voter data in firestore.
+        const resend = { dateTime: new Date().getTime(), reason: resendReason }
+        await firestoreService
+            .db
+            .collection('StateInfo')
+            .doc(info.id)
+            .update({
+                resends: admin.firestore.FieldValue.arrayUnion(resend)
+            })
+    }
+}
+
+// Reads './data/conditions.ts' and resends emails to all voter sign ups that satisfy query.
+const main = async () => {
+    // Fetch registrations.
+    const query: Query = newYorkQuery
+    const registrations = await firestoreService.query<RichStateInfo>(query)
+    if (registrations == null || registrations.length == 0) {
+        console.log('No voters found for specified conditions.')
+        return
+    }
+
+    // Display registration previews.
+    const registrationsToDisplay = registrations.length > 10 ? registrations.slice(0, 10) : registrations
+    console.log('There are a total of %s entries for your query.  Here is a sample:', registrations.length)
+    for (const registration of registrationsToDisplay) {
+        const previewMap = new Map(Object.entries(registration))
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        const previewToShowMap: Map<string, any> = new Map<string, any>()
+        previewToShowMap.set('id', registration.id)
+        for (const field of previewFields) {
+            const stringCondition: string = field
+            previewToShowMap.set(field, previewMap.get(stringCondition))
+        }
+        console.log(previewToShowMap)
+    }
+
+    // Get confirmation form user that emails should be sent.
+    const confirmation = 'Would you like to resend emails to all of these voters and their election officials? [\'Y\' | \'N\']'
+    const standardInput = process.stdin
+    standardInput.setEncoding('utf-8')
+    console.log(confirmation)
+    standardInput.on('data', async (data) => {
+        if (data.toString() != 'Y\n'){
+            console.log('You typed something other than \'Y\'. Exiting.')
+            process.exit()
+        } else {
+            await resendSignups(registrations, reason)
```

Comment:
You should probably `console.log` an acknowledgment here and then a success.  It's a little scary to see nothing.